rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -100,8 +101,8 @@ def persist(self, model_dir):
         # type: (Text) -> Dict[Text, Any]
         file_name = self.name+".json"
         full_name = os.path.join(model_dir, file_name)
-        with io.open(full_name, 'w') as f:
-            f.write(json.dumps({"dimensions": self.dimensions}))
+        with io.open(full_name, 'w', encoding="utf-8") as f:
```

Comment:
Although your fix is fine as well, the version I posted is just the one we used at other places and I'd like to keep it consistent (e.g. https://github.com/golastmile/rasa_nlu/blob/master/rasa_nlu/extractors/entity_synonyms.py#L63).